packages/uniforms/src/SimpleSchemaBridge.js

```diff
@@ -202,8 +202,8 @@ export default class SimpleSchemaBridge extends Bridge {
             } else {
                 field = {
                     ...field,
-                    transform: value => options.find(option => option.value === value).label,
-                    allowedValues: options.map(option => option.value)
+                    transform: value => options.find(option => option.value.toString() === value).label,
+                    allowedValues: options.map(option => option.value.toString())
```

Comment:
@radekmie I knew that, i need help in the way to do it. 

Right now i had to change the options and make the "toString" of all the data that is passed to the field. But i don't know hot to fix it in the antd selectfield